ivy/functional/ivy/manipulation.py

```diff
@@ -747,12 +747,90 @@ def constant_pad(
         axes of x.
     value
         The constant value to pad the array with.
+    out
+        optional output array, for writing the result to. It must have a shape that the
+        inputs broadcast to.
 
     Returns
     -------
     ret
         Padded array of rank equal to x with shape increased according to pad_width.
 
+
+    Functional Examples
+    -------------------
+    With :code:`ivy.Array` input:
+    >>> x = ivy.array([1, 2, 3, 4])
+    >>> y = ivy.constant_pad(x, (2, 3), (5, 6))
+    >>> print(y)
+    ivy.array([5 5 1 2 3 4 6 6 6])
+    >>> x = ivy.array([[1],[2],[3]])
+    >>> y = ivy.constant_pad(x, (1,2), (4,5))
+    >>> print(y)
+    ivy.array([[4, 4, 5, 5],
+       [4, 1, 5, 5],
+       [4, 2, 5, 5],
+       [4, 3, 5, 5],
+       [4, 5, 5, 5],
+       [4, 5, 5, 5]])
+    >>> x = ivy.array([[[1, 2],[3, 4]]])
+    >>> y = ivy.constant_pad(x, (1,2), (4,5))
+    >>> print(y)
+    ivy.array([[[4, 4, 4, 5, 5],
+        [4, 4, 4, 5, 5],
+        [4, 4, 4, 5, 5],
+        [4, 5, 5, 5, 5],
+        [4, 5, 5, 5, 5]],
+       [[4, 4, 4, 5, 5],
+        [4, 1, 2, 5, 5],
+        [4, 3, 4, 5, 5],
+        [4, 5, 5, 5, 5],
+        [4, 5, 5, 5, 5]],
+       [[4, 4, 4, 5, 5],
+        [4, 5, 5, 5, 5],
+        [4, 5, 5, 5, 5],
+        [4, 5, 5, 5, 5],
+        [4, 5, 5, 5, 5]],
+       [[4, 4, 4, 5, 5],
+        [4, 5, 5, 5, 5],
+        [4, 5, 5, 5, 5],
+        [4, 5, 5, 5, 5],
+        [4, 5, 5, 5, 5]]])
+    
+    With :code:`ivy.NativeArray` input:
+    >>> x = ivy.NativeArray([1, 2, 3, 4])
+    >>> y = ivy.constant_pad(x, (2, 3), (5, 6))
+    >>> print(y)
+    ivy.array([5 5 1 2 3 4 6 6 6])
+
+    With :code:`ivy.Container` input:
+    >>> x = ivy.Container(a=ivy.array([1, 2, 3]), \
+                          b=ivy.array([4, 5, 6]))
+    >>> y = ivy.constant_pad(x, (2, 3), (5, 6))
+    >>> print(y)
+    {
+        a: ivy.array([5 5 1 2 3 6 6 6]),
+        b: ivy.array([5 5 4 5 6 6 6 6])
+    }
+
+    Instance Method Examples
+    ------------------------
+    Using :code:`ivy.Array` instance method:
+    >>> x = ivy.array([1., 2., 3., 4.])
+    >>> y = ivy.constant_pad(x, (2, 3), (5., 6.))
+    >>> print(y)
+    ivy.array([5 5 1 2 3 4 6 6 6])
+
+    Using :code:`ivy.Container` instance method:
+    >>> x = ivy.Container(a=ivy.array([1., 2., 3.]), \
+                          b=ivy.array([4., 5., 6.]))
+    >>> y = ivy.constant_pad(x, (2, 3), (5., 6.))
+    >>> print(y)
+    {
+        a: ivy.array([5. 5. 1. 2. 3. 6. 6. 6.]),
+        b: ivy.array([5. 5. 4. 5. 6. 6. 6. 6.])
+    }
+
```

Comment:
Hi @Klins101, the formatting guide has changed a little bit. We no longer add all examples in the ivy docstring, rather we add it to their implementation specific docsrings. For example,  ``` instance method examples ``` for container will go in the ```container/manipulation/constant_pad``` function docstring. Kindly refer to the guide [here](https://lets-unify.ai/ivy/deep_dive/11_docstring_examples.html).

Also, while calling an instance method on an ```array``` or a  ```container```, we pass in that value as self and call the function with a ```dot``` operator. For example -:
```
x = ivy.array([1., 2., 3., 4.])
y = x.constant_pad((2,3), (5,6))

```

Kindly refer to the changes above 🙂

Reply:
Noted with thanks. I've made the required changes.